gradle/dependencies.gradle

```diff
@@ -49,6 +49,14 @@ ext.libraries = [
                     force = true
                 },
         ],
+        cassandra               : dependencies.create("com.datastax.cassandra:cassandra-driver-core:$cassandraDriverVersion") {
```

Comment:
Seems like this block should be merged into https://github.com/apereo/cas/blob/master/gradle/dependencies.gradle#L36?